bindings/cs/MongoDB.Libmongocrypt/Library.cs

```diff
@@ -442,11 +465,20 @@ internal class Delegates
             [return: MarshalAs(UnmanagedType.I1),]
             public delegate bool mongocrypt_setopt_schema_map(MongoCryptSafeHandle handle, BinarySafeHandle schema);
 
+            public delegate void mongocrypt_setopt_append_csfle_search_path(MongoCryptSafeHandle handle, [MarshalAs(UnmanagedType.LPStr)] string path);
+
+            public delegate void mongocrypt_setopt_set_csfle_lib_path_override(MongoCryptSafeHandle handle, [MarshalAs(UnmanagedType.LPStr)] string path);
+
             [return: MarshalAs(UnmanagedType.I1)]
             public delegate bool mongocrypt_init(MongoCryptSafeHandle handle);
 
             public delegate void mongocrypt_destroy(IntPtr ptr);
 
+            [return: MarshalAs(UnmanagedType.LPStr)]
+            public delegate string mongocrypt_csfle_version_string(MongoCryptSafeHandle handle, IntPtr length);
```

Comment:
copy from slack: did you investigate a difference between: mongocrypt_t *crypt  and const mongocrypt_t *crypt ?

Reply:
note: few times already I planned to start adding a full c file description to delegates here (in xml comments). It will be much easier to work here, but with no luck from my side yet. See whether you want start doing it or no :)